sentry/data_source_sentry_team_test.go

```diff
@@ -0,0 +1,39 @@
+package sentry
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccSentryTeamDataSource_basic(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccSentryTeamataSourceConfig,
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("data.sentry_team.test", "name", "Test team"),
+					resource.TestCheckResourceAttr("data.sentry_team.test", "slug", "test-team"),
+					resource.TestCheckResourceAttrSet("data.sentry_team.test", "team_id"),
+				),
+			},
+		},
+	})
+}
+
+// Testing first parameter
+var testAccSentryTeamataSourceConfig = fmt.Sprintf(`
+resource "sentry_team" "test_team" {
+  organization = "%s"
+  name = "Test team"
+	slug = "test-team"
+}
+
+data "sentry_team" "test_key" {
+  organization = "%s"
+	slug = "test-team"
+}
+`, testOrganization, testOrganization)
```

Comment:
This test is failing.
1. Fix typo
2. Full resource address should be `data.sentry_team.test_team`
3. Add `depends_on` to `data.sentry_team.test_team`

```suggestion
func TestAccSentryTeamDataSource_basic(t *testing.T) {
	resource.Test(t, resource.TestCase{
		PreCheck:  func() { testAccPreCheck(t) },
		Providers: testAccProviders,
		Steps: []resource.TestStep{
			{
				Config: testAccSentryTeamDataSourceConfig,
				Check: resource.ComposeTestCheckFunc(
					resource.TestCheckResourceAttr("data.sentry_team.test_team", "name", "Test team"),
					resource.TestCheckResourceAttr("data.sentry_team.test_team", "slug", "test-team"),
					resource.TestCheckResourceAttrSet("data.sentry_team.test_team", "team_id"),
				),
			},
		},
	})
}

// Testing first parameter
var testAccSentryTeamDataSourceConfig = fmt.Sprintf(`
resource "sentry_team" "test_team" {
  organization = "%s"
  name = "Test team"
  slug = "test-team"
}

data "sentry_team" "test_team" {
  organization = "%s"
  slug = "test-team"
  depends_on = [
    sentry_team.test_team
  ]
}
`, testOrganization, testOrganization)
```